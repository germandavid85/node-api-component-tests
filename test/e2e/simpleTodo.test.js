const fetch = require('node-fetch');

describe('simple todo', () => {
  let allTheNotes;

  beforeAll(async () => {
    const response = await fetch('http://localhost:3000/todos');

    allTheNotes = await response.json();
  });

  test('should return all the todos', () => expect(allTheNotes).toMatchSnapshot());
});
