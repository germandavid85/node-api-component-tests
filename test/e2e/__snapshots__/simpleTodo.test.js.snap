// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple todo should return all the todos 1`] = `
Array [
  Object {
    "createdAt": "Thu Jul 02 2020 18:46:22 GMT-0500 (Colombia Standard Time)",
    "ready": "false",
    "title": "this is a normal todo",
    "updatedAt": "Thu Jul 02 2020 18:46:22 GMT-0500 (Colombia Standard Time)",
  },
  Object {
    "createdAt": "Thu Jul 02 2020 18:46:34 GMT-0500 (Colombia Standard Time)",
    "ready": "false",
    "title": "w. this is a watson todo",
    "updatedAt": "Thu Jul 02 2020 18:46:34 GMT-0500 (Colombia Standard Time)",
  },
]
`;
